### ISSUE TITLE
Visual label of a component contains text associated with it programmatically

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/structured-metadata-editor-react#74]
### Description
SME audit report line item 2.5.3, referenced in this issue:  #64 

WCAG 2.1 Checkpoint for Label in Name - Each list item within the structure tree does not have an accessible label to identify them associated with their name

### Done Looks Like
- Each list item has a label which can be used to identify them properly

### To Do
- [ ] Implement a solution
- [ ] Re-test.

This is covered in #73
